### ISSUE TITLE
[Can you push FCN8 code]

### ISSUE THREAD

**Issue by Vietdung113** [HarisIqbal88/PlotNeuralNet#48]
Thank you for your publish code. Can you add sample of FCN or shorcut connection code?

**Comment by luotuoqingshan** [comment]
I read the examples carefully. I think the latex version is easy to understand too. You can use latex directly.

**Comment by beaniubideren** [comment]
I also need, especially how to program the addition of the two outputs！！！

**Comment by AlexandreBourrieau** [comment]
Yes please post the code to create skip connexions with pyton which are on a connexion...

**Comment by cwreynolds** [comment]
Thanks for this utility from 2018. The diagrams are nice, but the instructions for use are rather cryptic.

Is the intent primarily to show some examples of how to use LaTeX to generate diagrams of neural nets? If so, is the idea I should pick one in PlotNeuralNet/examples/ most like my net, then modify its LaTeX code by hand to get what I need?

And if so, why does **Getting Started** (in README) seem to assume it will be used from Python? And if it **is** supposed to be used from Python, why is there no Python code in PlotNeuralNet/examples/ 

My assumption is that hand-modifying the LaTeX examples, ignoring Python, is the best way to go. But as a LaTeX newbie, before I go down that path I was hoping for reassurance that was the correct approach.

**Comment by cwreynolds** [comment]
Just in case anyone else comes across my previous comment: that is what I did, copied one of the examples at Overleaf, then modified it to meet my needs.

This was slow going at first since I am new to LaTeX, and knew nothing of tikz, but eventually puzzled out what was going on. Using this copy-and-modify approach (versus say generating the LaTeX from python) makes it easy to customize the graphics by simply modifying my local copy of RightBandedBox.sty 

EG for my diagram, I turned off the “Banded-ness” (by commenting out the code that drew it) and changed the placement for xlabel, ylabel, and zlabel relative to the oblique boxes, as mentioned here: https://tex.stackexchange.com/questions/475055/how-to-align-the-label-to-centre#comment1654334_484098
